### DISCUSSION TITLE
Share information around split block tools

### DISCUSSION THREAD

**Discussion by annezazu** [WordPress/developer-blog-content#57] [Topic Ideas]
The split block experiment was stabilized and merged recently, aligning it for the 6.2 release: https://github.com/WordPress/gutenberg/issues/40204

As a result, I think we need to create an informational post around how this might impact any third party folks as you can see here in this convo with Nick Diego in the PR: 
https://github.com/WordPress/gutenberg/pull/45005#issuecomment-1352823904

Would be great to get a post out ahead of/during beta to encourage feedback too!

**Comment by bph** [comment]
How would, could or should a  post like differe from a [Release Dev Note] (https://github.com/WordPress/gutenberg/pull/47045#issuecomment-1379782062)? 

On first glance this seems to be duplicating content, one of them major concerns in the proposal phase of the blog. And it's purpose was that's in addition to the Make blog and not a replacement. 
I can see that after the release a tutorial on how to use the sidebar for plugin purposes would be useful and we might need to review the [Slottfill post by Ryan](https://developer.wordpress.org/news/2022/11/how-to-extend-wordpress-via-the-slotfill-system/) to update it for the split display.

**Reply by annezazu** [reply]
In my mind, a post on the developer blog could go into more detail with various examples and offer more guidance for how developers should build with this change in mind. If it feels like too much duplication, totally cool to close out!

**Reply by bph** [reply]
To form a good opinion, I wait for the dev note and continue the discussion afterwards.

**Comment by bph** [comment]
The [tracking issue for 6.2 dev notes](https://github.com/WordPress/gutenberg/issues/47771) is a work in progress..

**Comment by justintadlock** [comment]
This has already come up on the Developer Blog as a comment: https://developer.wordpress.org/news/2023/02/creating-custom-block-styles-in-wordpress-themes/#comment-51

I think the biggest thing devs might be asking is exactly that question:  _How do I specifically assign controls to one or the other tab?_

I'd like to see what the dev note would be, but I could see a post on the Dev Blog showcasing a real example of how to do this.

**Comment by ndiego** [comment]
Depending on how thorough the Dev Note is, I would be happy to pick this one up. I am already in the process of implementing this functionality in my block plugins.

**Reply by ndiego** [reply]
Here's an early draft of the dev note: https://github.com/WordPress/gutenberg/pull/47045#issuecomment-1425428525

**Comment by bph** [comment]
Seems that was already covered in Ryan's post: 
[Using block inspector sidebar groups](https://developer.wordpress.org/news/2023/06/using-block-inspector-sidebar-groups/)
I close this discussion. If I am misunderstanding something essential here, we can certainly reopen.
